src/core/logging_manager.py

```diff
@@ -0,0 +1,129 @@
+"""Logging Manager.
+
+This module provides classes to represent logging configuration.
+"""
+
+import json
+import logging
+import os
+import sys
+from datetime import datetime
+
+import colorlog
+
+from src.core.constants import CommonPaths
+
+
+class PrintHandler(logging.StreamHandler):
+    """Custom Print handler."""
+
+    def emit(self, record):
+        """Custom print handler.
+
+        :param record:
+        :return:
+        """
+        msg = self.format(record)
+        print(msg)
+        sys.stdout.flush()
+        sys.stderr.flush()
+
+
+class CustomJsonFormatter(logging.Formatter):
+    """Custom JSON formatter."""
+
+    def format(self, record: logging.LogRecord):
+        """Custom json format for log message.
+
+        :param record:
+        :return:
+        """
+        log_message = {
+            "datetime": datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
+            "name": record.name,
+            "level": record.levelname,
+            "filename": f"{record.filename}:{record.lineno}",
+            "message": record.getMessage(),
+        }
+        return json.dumps(log_message)
+
+
+class LoggingManager:
+    """Custom Loging Manager."""
+
+    _LOG_FILE_NAME = "appointment-scheduler.log"
+
+    @staticmethod
+    def init_logger_json_format():
+        """Setup logger in custom JSON format.
+
+        :return:
+        """
+        logger = logging.getLogger(__name__)
+        logger.setLevel(logging.DEBUG)
+
+        log_format = "%(log_color)s%(asctime)s - %(name)s - [%(levelname)s] -  %(filename)s:%(lineno)d - %(message)s"
```

Comment:
@vasylboliuk looks like you combine several separate actions into one huge function. Please decouple formatting, add color formatting, add file handler and other handlers to separate methods. We want our code to follow Single responsibility principle